pkg/services/cwe.go

```diff
@@ -0,0 +1,59 @@
+package services
+
+import (
+	"embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/kptm-tools/vulnerability-analysis/pkg/dto"
+)
+
+type CWEInformation struct {
```

Comment:
If this is to be loaded from `main.go` once and then injected somewhere else, we should respect the naming we've been using for services in this repo:

```suggestion
type CWEService struct {
```

Reply:
solved